src/index.js

```diff
@@ -110,6 +110,11 @@ class Switch extends EE {
     this.stats.stop()
     series([
       (cb) => each(this.muxedConns, (conn, cb) => {
+        // If the connection was destroyed while we are hanging up, continue
+        if (conn === undefined) {
```

Comment:
Should this check be so restrictive? Perhaps an `if (conn) {...}` should do?

Reply:
Update pushed, this should be good to go.